src/ImageService.Business/Commands/ImageNews/GetImageNewsCommand.cs

```diff
@@ -1,45 +1,51 @@
-﻿using LT.DigitalOffice.ImageService.Business.Commands.ImageNews.Interfaces;
+﻿using System;
+using System.Net;
+using LT.DigitalOffice.ImageService.Business.Commands.ImageNews.Interfaces;
 using LT.DigitalOffice.ImageService.Data.Interfaces;
 using LT.DigitalOffice.ImageService.Mappers.Responses.Interfaces;
 using LT.DigitalOffice.ImageService.Models.Db;
 using LT.DigitalOffice.ImageService.Models.Dto.Responses;
 using LT.DigitalOffice.Kernel.Enums;
 using LT.DigitalOffice.Kernel.Responses;
-using System;
+using Microsoft.AspNetCore.Http;
 
 namespace LT.DigitalOffice.ImageService.Business.Commands.ImageNews
 {
-    public class GetImageNewsCommand : IGetImageNewsCommand
+  public class GetImageNewsCommand : IGetImageNewsCommand
+  {
+    private readonly IImageNewsRepository _imageNewsRepository;
+    private readonly IImageDataResponseMapper _imageNewsResponseMapper;
```

Comment:
still don't know why you don't want to rename ImageDataResponse to ImageResponse and other classes respectively

Reply:
Done